src/rbac-forms/rbacUserManagementModule/index.jsx

```diff
@@ -0,0 +1,137 @@
+import React, { PureComponent } from 'react';
+import { Provider, connect } from 'react-redux';
+import { Route, Switch } from 'react-router-dom';
+import { ConnectedRouter, goBack } from 'connected-react-router';
+import { Button, Grid, Row, Col, ListGroup, ListGroupItem, Spinner } from 'patternfly-react';
+import PropTypes from 'prop-types';
+import { history, store } from './redux/reducers/';
+import { navigate, requestUsers, deleteUser } from './redux/actions/actions';
+import UsersList from './pages/usersList';
+import UserAdd from './pages/userAdd';
+import UserEdit from './pages/userEdit';
+import UserPreview from './pages/userPreview';
+import { data } from './redux/data';
+
+const Toolbar = ({
+  handleAdd,
+  handleEdit,
+  canEdit,
+  editUserId,
+  handleDelete,
+}) => (
+  <div style={{ marginBottom: 10 }}>
+    <Button onClick={handleAdd}>Add</Button>
+    <Button disabled={!canEdit} onClick={() => handleEdit(editUserId)}>Edit selected</Button>
+    <Button disabled={!canEdit} onClick={() => handleDelete(editUserId)}>Delete selected</Button>
+  </div>
+);
+
+Toolbar.propTypes = {
+  handleAdd: PropTypes.func.isRequired,
+  handleEdit: PropTypes.func.isRequired,
+  canEdit: PropTypes.bool.isRequired,
+  editUserId: PropTypes.number,
```

Comment:
Shouldn't this be also required?

Reply:
Probably yes, but the index.jsx is just a demo file for storybook. This wrapping component should be implemented in MIQ and just provide data and callbacks for other components.